docs/reference/dlm/apis/explain-data-lifecycle.asciidoc

```diff
@@ -55,7 +55,7 @@ GET .ds-metrics-2023.03.22-000001/_lifecycle/explain
 --------------------------------------------------
 // TEST[skip:we're not setting up DLM in these tests]
 
-If the index is managed by DLM `explain` will show the `managed_by_dlm` field
+If the index is managed by DLM `explain` will show the `managed_by_lifecycle` field
```

Comment:
```suggestion
If the index is managed by the data stream lifecycle `explain` will show the `managed_by_lifecycle` field
```

Reply:
I believe this file contains more references to dlm - https://github.com/elastic/elasticsearch/blob/main/docs/reference/dlm/apis/explain-data-lifecycle.asciidoc?plain=1#L27
https://github.com/elastic/elasticsearch/blob/main/docs/reference/dlm/apis/explain-data-lifecycle.asciidoc?plain=1#L58

Reply:
@n1v0lg Can we rename the `manage_dlm` privilege to `manage_lifecycle` or so? https://github.com/elastic/elasticsearch/blob/main/docs/reference/dlm/apis/explain-data-lifecycle.asciidoc?plain=1#L16

Are there BWC concerns?

Reply:
The security rebranding could be done in a follow-up

Reply:
I was planning to address this in the [documentation PR](https://github.com/elastic/elasticsearch/pull/95326), in order to ensure I will be using consistent language in the tutorials and the APIs. Would that be ok?

Reply:
About rebranding security I added it as an extra task in https://github.com/elastic/elasticsearch/issues/96875

Reply:
I think we should be able to do this without BWC concerns: we introduced [the new privilege in 8.9](https://github.com/elastic/elasticsearch/pull/95512), and haven't removed the DLM feature flag yet, so changing it in 8.9 should be fine, without any extra work needed.

We just have to make sure we update raw string references to `manage_dlm` everywhere to the new privilege name. @gmarouli will you be able to pick this up?

Reply:
Yes let me do this right away, now that it's fresh in our minds :)